src/main/java/baritone/pathing/movement/movements/MovementParkour.java

```diff
@@ -44,23 +45,23 @@
 
 public class MovementParkour extends Movement {
 
-    private static final EnumFacing[] HORIZONTALS_BUT_ALSO_DOWN____SO_EVERY_DIRECTION_EXCEPT_UP = {EnumFacing.NORTH, EnumFacing.SOUTH, EnumFacing.EAST, EnumFacing.WEST, EnumFacing.DOWN};
+    private static final EnumFacing[] HORIZONTALS_BUT_ALSO_DOWN____SO_EVERY_DIRECTION_EXCEPT_UP  = { EnumFacing.NORTH, EnumFacing.SOUTH, EnumFacing.EAST, EnumFacing.WEST, EnumFacing.DOWN };
```

Comment:
<img width="88" alt="screen shot 2018-11-13 at 2 47 01 pm" src="https://user-images.githubusercontent.com/3837873/48448165-fd6e5100-e752-11e8-9792-a96582794863.png">

<img width="100" src="https://pics.me.me/que-es-en-tarnacion-15344882.png">

Reply:
LOL fixed.

The second space is because I don't want autism arrays.